backend/inspirehep/records/tasks.py

```diff
@@ -195,5 +196,12 @@ def remove_bai_from_literature_authors(uuids):
                 author["ids"] = new_ids
             else:
                 del author["ids"]
-        record.update(dict(record), disable_disambiguation=True)
-        db.session.commit()
+        try:
+            record.update(dict(record), disable_disambiguation=True)
+            db.session.commit()
+        except ValidationError:
```

Comment:
shall we catch all errors gracefully? otherwise it will blow-up all the time

Reply:
I don't know, I can't think of other errors that might appear and do not repeat the task